torch/_inductor/scheduler.py

```diff
@@ -360,6 +360,80 @@ def codegen_originating_info(self, buffer, only_once=True):
         buffer.writelines(out_lines)
         self.written = True
 
+    def get_estimated_runtime(self, rwbytes: int) -> float:
```

Comment:
This probably shouldn't take in the rwbytes and the code to get the read/write bytes should be refactored to be in scheduler.py

Reply:
Addressed in d1fb04c. `get_read_write_buffers_sizes` is computed twice, and it's a bit awkward to address. I'm calling `BaseSchedulerNode::get_estimated_runtime` from `GraphLowering::count_bytes` just to have a unit test cover it.

For prod, I'm expecting `get_estimated_runtime` to be called from elsewhere e.g. https://github.com/pytorch/pytorch/pull/100762, and those code paths will not need `get_read_write_buffers_sizes`, so we shouldn't have it as an output argument of `BaseSchedulerNode::get_estimated_runtime` just to avoid computing it a second time